vendor/vendor-android/src/main/kotlin/com/malinskiy/marathon/android/DdmlibDeviceExtensions.kt

```diff
@@ -94,4 +116,17 @@ fun getScreenRecorderCommand(@NonNull remoteFilePath: String,
     sb.append(remoteFilePath)
 
     return sb.toString()
-}
\ No newline at end of file
+}
+
+class SimpleOutputReceiver: MultiLineReceiver() {
+    private val buffer = StringBuffer()
```

Comment:
Do we need StringBuffer here or it can be replaced with StringBuilder? StringBuilder is not safe for multithreaded usage. I'm not sure, but probably this received will not be used from multiple threads

Reply:
I'm not sure. Supposedly Ddmlib should call it from single thread, but I don't trust it